openpype/settings/defaults/project_settings/nuke.json

```diff
@@ -341,7 +341,7 @@
                 "write"
             ]
         },
-        "ValidateCorrectAssetName": {
+        "ValidateCorrectAssetContext": {
```

Comment:
since you are here you might as well update the `ValidateScript` setting, it should be `ValidateScriptAttributes`

Reply:
lets do this change in https://github.com/ynput/OpenPype/pull/5746